.storybook/webpack.config.js

```diff
@@ -89,6 +91,7 @@ if (USER_ID && USER_ACCESS_TOKEN) {
   plugins.push(
     new webpack.DefinePlugin({
       "process.env": {
+        IS_STORYBOOK: JSON.stringify(true),
```

Comment:
?

Reply:
Related to comment below. We need a way to check that we're inside of storybooks during runtime in order to properly navigate inside of its iframe.